app/controllers/bulk_actions_controller.rb

```diff
@@ -14,21 +14,16 @@ def index
 
   # GET /bulk_actions/new
   def new
-    @bulk_action = BulkAction.new
+    @form = BulkActionForm.new(BulkAction.new, groups: current_user.groups)
```

Comment:
I wonder whether the `BulkActionForm` ought to be responsible for instantiating a `BulkAction`, given a user instance. From that user instance, the form can create the instance and get the groups. I don't think it's a big deal, though.

Reply:
I don't want to do that because it changes the pattern that all forms use (and the pattern that reform uses)

Reply:
Makes sense.